### DISCUSSION TITLE
Is tide supposed to be used in conjunction with projectile

### DISCUSSION THREAD

**Discussion by steveeq1** [ananthakumaran/tide#405] [Q&A]
Forgive the naive question, but should tide work in conjunction with projectile? Or are you supposed to use one, but not the other?

**Comment by josteink** [comment]
I've used both tide and projectile at the same time for as long as I can remember, and it works fine right out of the box. 

May I ask why you're asking such a question?

**Comment by steveeq1** [comment]
I am learning tide and I want to know what the "best practices" are. I didn't know if one replaced the other.

**Comment by josteink** [comment]
They work together. Need to navigate to symbol under cursor? Use tide. 

Need to navigate to random file which is not in a immediate tide/typescript context (or just html or whatever)? Use projectile. 

They both have their place.
